toxcore/tox.api.h

```diff
@@ -2321,6 +2321,70 @@ namespace conference {
 
   }
 
+  namespace offline_peer {
+
+    /**
+     * Error codes for peer info queries.
+     */
+    error for query {
```

Comment:
error for peer.query can be reused (I hope that there is a syntax for that that works).